odoo/modules/registry.py

```diff
@@ -295,10 +298,14 @@ def setup_models(self, cr, partial=False):
         for model in models:
             model._setup_complete()
 
+    def post_init(self, func, *args, **kwargs):
+        """ Register a function to call at the end of :meth:`~.init_models`. """
+        self._post_init_queue.append(partial(func, *args, **kwargs))
```

Comment:
Maybe just take a callable and if the caller wants to pass in args they can partially apply it themselves?